### PR TITLE
First version of a flowcells list for Element BioSciences runs

### PR DESCRIPTION
**Pull request by alneberg**
- A new tab on the flowcells page for Element Biosciences (AVITI) runs.
 - Skeleton code for a page showing individual runs (to keep the PR small)

### CODE REVIEW

File: run_dir/design/element_flowcells.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            </tr>
+        </tfoot>
+        <tbody class="list">
+            {% for onefc in element_fcs %}
+                <tr>
+                    <td class="start_date">{{ onefc.key }}</td>
+                    <td class = run_name>
+                        <a class="text-decoration-none" href="/flowcells_element/{{ onefc.id }}">{{ onefc.value['RunName'] }}</a>
+                        {% if onefc.value.get('outcome') == 'OutcomeCompleted' %}
+                            <abbr title="Run completed"><span class="fa fa-check-circle"></span></abbr>
```
Comment (kedhammar) [inline run_dir/design/element_flowcells.html:42]: I don't think the fa-symbols are showing up?

File: run_dir/design/element_flowcells.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                <th class="sort searchable" data-sort="throughput_selection">Throughput Selection</th>
+                <th class="sort searchable" data-sort="kit_configuration">Kit Configuration</th>
+                <th class="sort searchable" data-sort="chemistry_version">Chemistry Version</th>
+                <th class="sort text-center" data-sort="outcome">Outcome</th>
+            </tr>
+        </tfoot>
+        <tbody class="list">
+            {% for onefc in element_fcs %}
+                <tr>
+                    <td class="start_date">{{ onefc.key }}</td>
```
Comment (kedhammar) [inline run_dir/design/element_flowcells.html:38]: I'd prefer if this followed the same as Illumina and ONT i.e. yyyy-mm-dd

**Review by kedhammar — APPROVED** [review]
Good start start, just a few comments that can be addressed at any point

File: run_dir/design/element_flowcells.html (hunk 2 above)
Comment (alneberg) [reply run_dir/design/element_flowcells.html]: I agree. I feel dirty for the fix I did, but it should work for the next 8000 years.

File: run_dir/design/element_flowcells.html (hunk 1 above)
Comment (alneberg) [reply run_dir/design/element_flowcells.html:42]: Good catch! It was a typo and should be fixed now.

**Pull request merged**